domain/api/atrium-domain-api-common/src/main/kotlin/ch/tutteli/atrium/domain/creating/ComparableAssertions.kt

```diff
@@ -29,4 +29,9 @@ interface ComparableAssertions {
         subjectProvider: SubjectProvider<T1>,
         expected: T2
     ): Assertion
+
+    fun <T1 : Comparable<T2>, T2 : Any?> isEqualComparingTo(
+        subjectProvider: SubjectProvider<T1>,
```

Comment:
please use Expect, SubjectProvider will most likely be removed - was only introduced for migration purposes
```suggestion
        expect: Expect<T1>,
```